frontend/packages/kubevirt-plugin/src/topology/components/kubevirtComponentFactory.ts

```diff
@@ -20,20 +20,22 @@ import {
   TopologyDataObject,
   getTopologyResourceObject,
 } from '@console/dev-console/src/components/topology';
-import { ExtraResources, vmMenuActionsCreator } from '../../components/vms/menu-actions';
+import { vmMenuActions } from '../../components/vms/menu-actions';
 import { VmNode } from './nodes/VmNode';
 import { TYPE_VIRTUAL_MACHINE } from './const';
-import { VMKind } from '../../types/vm';
 
 export const vmActions = (vm: TopologyDataObject): KebabOption[] => {
   const contextMenuResource = getTopologyResourceObject(vm);
   if (!contextMenuResource) {
     return null;
   }
-  const data = vm.data as ExtraResources;
+  const data = vm.data as any;
```

Comment:
why `as any` ? Cant we define something like `ExtraResources & { statusDetail: VMStatusBundle }` ? We should also use generics here https://github.com/openshift/console/blob/bc92498ec320ff54a7e6c7c1e7179bf53846218b/frontend/packages/kubevirt-plugin/src/topology/components/kubevirtComponentFactory.ts#L27

```
vm: TopologyDataObject<ExtraResources & { statusDetail: VMStatusBundle }>
```

Reply:
+1, I had to commit a bit more changes to fix the types